examples/Example 1.yml

```diff
@@ -0,0 +1,58 @@
+# Teams Section
+Teams:
+- Name: Team 1
+  Identity: Security Group 1
+- Name: Team 2
+  Identity: Security Group 2
+
+# Patterns
+Patterns:
+- !Security # Exclamation mark is not actual yaml, but the yaml library understands it as inheritance, parent is pattern
+  Name: Default
+  Application Groups:
+  - Name: Contributor
+    Members: Security Group {0} # {0} will be replaced with a key using a string.format
+    Namespaces:
+    - Name: Project
+      Allow:
+      - Create tag definition
+      - Create test runs
+      - Edit build quality
+      - Edit project-level information
+      - Manage project properties
+      - Manage test configurations
+      - Manage test environments
+      - Move work items out of this project
+      - Start a build
+      - Suppress notifications for work item updates
+      - View analytics
+      - View project-level information
+      - View test runs
+      - Write to build operational store
+      Deny:
+      - Bypass rules on work item updates
+      - Change process of team project.
+      - Delete and restore work items
+      - Delete team project
+      - Delete test runs
+      - Update project visibility
+      - Write to build operational store
+      Not Set:
+      - ...
```

Comment:
If Not set is defined, what will we do with something that is missing in the list ?

Reply:
If we do not add it to the list, it will do nothing. But actually I was doubting "not set"

Reply:
Should the patterns be specified in a separate file?

Reply:
As the patterns have a name, we could advice to put the versioning in the name. Would that be sufficient. Regarding a seperate file I agree.